src/InputFieldComponent.js

```diff
@@ -32,7 +32,7 @@ const InputField = (props) => {
 const numbers = [1,2,3,4,5,6,7,8,9,10]
 const instructions = [ 
```

Comment:
may be worth making these bold for emphasis (not worth spending lots of time on)